bin/cmu_auth

```diff
@@ -0,0 +1,28 @@
+#!/Users/jake/.virtualenvs/cmuauth-test/bin/python2.7
```

Comment:
Sorry, that's my bad. I had been testing it locally, made some changes, and copied it over.